### DISCUSSION TITLE
not able to start Winlogbeat-Oss 8.7.1

### DISCUSSION THREAD

**Discussion by homayoonfayaz2** [Security-Onion-Solutions/securityonion#10625] [Unsupported Versions]
after installing the agent I tried to start the services but i faced with below error

Windows could not start the Elastic Winlogbeat-Oss 8.7.1 service on Local Computer
Error 1053: The service did not respond to start or control request in a timely fashion

**Comment by cm-ops** [comment]
Have you tried to uninstall WLB and then reinstall it to see if that fixes the issue?

**Reply by homayoonfayaz2** [reply]
yeah i tried but it's not working

**Comment by InfosecGoon** [comment]
Did you configure the winlogbeat.yml file in C:\ProgramData\Elastic\Beats\winlogbeat? If you don't have a proper configuration in there the service won't start.

**Reply by homayoonfayaz2** [reply]
It should be configured properly as I clone it from github

**Reply by InfosecGoon** [reply]
Confirm that the Elasticsearch output is commented out in the configuration file, and that the Logstash output is configured with the IP or hostname of your Security Onion node.

**Reply by homayoonfayaz2** [reply]
Thanks for your reply can you please elaborated more

**Reply by homayoonfayaz2** [reply]
Thanks the service get started but still the logs are not showing in the security onion dashboard

**Reply by homayoonfayaz2** [reply]
Thanks the service get started but still the logs are not showing in the security onion dashboard

**Reply by InfosecGoon** [reply]
Did you use so-allow to open a firewall hole on the Manager node to listen for connections from the computer running winlogbeat? You need to run "sudo so-allow", then option B (for Beats), then the IP or subnet of the host that's sending logs.

**Reply by homayoonfayaz2** [reply]
Thanks for your support but I did that already

**Reply by InfosecGoon** [reply]
Okay, check the Winlogbeat logs on the sending computer (C:\ProgramData\Elastic\Beats\winlogbeat\logs) and the Logstash logs on the receiving node (/opt/so/log/logstash/logstash.log) for errors.
